pkg/compose/start.go

```diff
@@ -111,7 +112,23 @@ func (s *composeService) watchContainers(ctx context.Context, projectName string
 			}
 			name := getContainerNameWithoutProject(container)
 
+			if event.Status == "stop" {
+				listener(api.ContainerEvent{
+					Type:       api.ContainerEventExit,
```

Comment:
In the end it should make the same, but shouldn't it be stopped?
```suggestion
					Type:       api.ContainerEventStopped,
```

Reply:
oh yes, forgot to update this